components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -262,21 +253,12 @@ export default class CreateWorkspace extends React.Component<CreateWorkspaceProp
                         </>
                     </div>
                     <div className="flex justify-end mt-6">
-                        <button onClick={() => this.createWorkspace(CreateWorkspaceMode.Default)}>New Workspace</button>
+                        <button onClick={() => this.createWorkspace({ ignoreRunningWorkspaceOnSameCommit: true })}>
+                            New Workspace
+                        </button>
                     </div>
                 </Modal>
             );
-        } else if (result?.runningWorkspacePrebuild) {
-            return (
```

Comment:
Can we put this behind a feature flag, just to decouple it from the rollout of the "incremental workspace" feature?

Reply:
I could make the removal of `runningWorkspacePrebuild` a separate PR, that we merge first. But I'm not sure that buys us a lot. Would be super helpful to dive into the uncertainty a bit more and figure out if there is something we missed. I can't think of any issues that the removal of this feature could have on existing workflows or deployment.

Reply:
At the core, the concern I have is: What happens to a high-frequency repo, that did not enable incremental workspaces yet, once we roll this out? To me it feels they'd just never benefit from Prebuilds anymore. Until they understand that there is Incremental Workspaces, and it works for them. This feels like a bad spot for us to be in, and it feels unnecessary as well.

Alternatively, we could leave the screen as-is, maybe tied to `!opts.incrementalWorkspace`. That way, people can still try to manually fall back to "Use last successful prebuild" (current code) if they want to.
Plus it gives us an obvious spot where to out the advertisement for the "Incremental Workspaces" option, which people can learn about and "migrate" to. :lotus_position: